dev.schlaubi.Tonbrett.yml

```diff
@@ -0,0 +1,39 @@
+app-id: dev.schlaubi.Tonbrett
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+command: Tonbrett
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk
+  build-args:
+    - --share=network
+    - --filesystem=host
+finish-args:
+  - --share=network # required for auth server and to contact API
+  - --filesystem=home # required to store token
+  # required for AWT
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+modules:
+  - name: Tonbrett
+    buildsystem: simple
+    build-commands:
+      - ./gradlew :app:desktop:createDistributable
+      - cp app/desktop/build/compose/binaries/main/app/Tonbrett/* $FLATPAK_DEST -r
+      - install -Dm 644 -t /app/share/applications dev.schlaubi.Tonbrett.desktop
+      - install -Dm 644 logo.png /app/share/icons/hicolor/64x64/apps/dev.schlaubi.Tonbrett.png
+    sources:
+      - type: git
+        url: https://github.com/DRSchlaubi/tonbrett.git
+        tag: 1.9.6
+        x-checker-data:
+          type: git
+          tag-patern: ^([\d.]+)$
+      - type: file
+        path: logo.png
```

Comment:
you shoudln't need this ere since https://github.com/DRSchlaubi/tonbrett/blob/main/app/desktop/src/main/resources/logo.png exists.

Reply:
I originally used that directly, however this logo is an a different format

Reply:
ah yeah it's not square not is the size correct for an icon. maybe it is good occasion to upstream the icons and the .desktop properly.

Reply:
it is still possible.

Reply:
you can download individual files from a repo on github.

Reply:
I don't really want to do that if it can be avoided